Added Xiaomi Redmi Note 9S Overlay

sooti opened the pull request:
Fixed tilt issues and brightness

phhusson committed:
Remove in-tree messy apps, and switch to TrebleApp

phhusson committed:
Update TrebleApp apk as of 3d403df589310b5a74d88d7fec856948b43a115c

Changes:
 - Detect CAF IMS apk and enable RRO and send CHANGED intent to trigger it
 - Don't fail when writing fast charge
 - Add "release" build.sh target
 - Enable S9 systemui overlay on S9/Note9
 - Don't show samsung settings on non-samsung devices

phhusson committed:
Update TrebleApp to 658099e446c70a312a1cb7536038d2d903f2e5c0

phhusson committed:
Updatre Treble app to 70740a754ed89e11e126618ff8f1df28fc3330b5

phhusson committed:
Update Treble app to a91d94595e05270475b3188a3adce4656f2517aa

phhusson committed:
Delete hw-fpnav. This is now handled by Treble app

phhusson committed:
Update Treble app to 7eae005766ae4cd0e04209582d573ac94fd31eb1

phhusson committed:
Update Treble app to 1808b79a206251c2947e988501d63deb011cfc08

Commit:
Dehardcode status_bar_height for Pie

jackwakefield committed:
Update power_profile.xml for Xiaomi Mi A2

phhusson committed:
Forbid status_bar_height in pie

phhusson committed:
Remove status_bar_height in pie for moto one

phhusson committed:
Update Treble app to dfa8451d26fe375bd9e44b44503448761241eb70

phhusson committed:
[Mi9] Initial overlay for Mi 9

phhusson committed:
[Mi9] Allow auto brightness while dozing; make brightness ramps much faster to fix under-display FP behaviour

phhusson committed:
Update Treble app to 2c8c7615baa639676180d7d2423533f22b45e383

phhusson committed:
Remove status_bar_height for Tecno Camon11

phhusson committed:
Update Treble app to c2818a35039d0e37e5a8f83a9a4ef5ee8828546b

phhusson committed:
De-hardcode status_bar_height

AndyCGYan committed:
Add resources for Samsung Galaxy Tab A 10.1 (2019, gta3xl)

Change-Id: I44f386c500efb6d8eba21b81293b5973433ca960

phhusson committed:
[phh-overrides] add few more gapps to be ignored

phhusson committed:
Don't remove WebViewGoogle on PE gapps

phhusson committed:
Partial revert of d402d8d. AOSP variants arent installed either

phhusson committed:
Use proper folder when giving an argument

phhusson committed:
Add overlay for Nokia 4.2

phhusson committed:
Update treble app to 7d67b6760ae01e2bb816865425d6ee0700d4b794

phhusson committed:
De-hardcode status_bar_height in zenfone max m2 overlay

phhusson committed:
Update TrebleApp to eb0bebe4c2d4acd3835f63daa6bb96a35153a11f

phhusson committed:
De-hardcode status_bar_height in zenfone max m2 overlay

phhusson committed:
Update Treble App to a88ed09136b14dc0f781f7d04c85edb60270ca9f

phhusson committed:
S10e has a android.sensor.pick_up_gesture sensor

PeterCxy committed:
add Qin 2 Pro from DuoQin

AndyCGYan committed:
Add resources for Samsung Galaxy A60 (a60q) / M40 (m40)

Change-Id: Ifb47912db30c1491b47b556fc2da94ff80c57e0c

AndyCGYan committed:
Remove M40 compatibility from A60 overlay

Glob pattern doesn't work, and this has never been tested on M40 anyway

Change-Id: Ib2efeb83fcafb8c7ca1c603bcd797acac7224cdd

phhusson committed:
Fix priority conflict

phhusson committed:
Update treble app to 94fbc1fe42b66a024cd1c456ed0bdac1b3399568

phhusson committed:
Update TrebleApp to 366ae35

phhusson committed:
Updat treble app to 005ad78

phhusson committed:
[Realme X2 Pro] More reasonable minimum brightness (issue is the scale 10bit)

phhusson committed:
Gain 81MB by removing... sound recorder? what?

phhusson committed:
Shrrrrinnnkkk

phhusson committed:
WebviewGoogle is still needed, sadly PE dont have the lightweight one using trichrome

phhusson committed:
Add turboprebuilt to gapps removes

phhusson committed:
Update treble app to 05c1b5f0

phhusson committed:
Update treble app to 8d6ff5e

phhusson committed:
Up treble app to 28ee70

sooti committed:
Added Xiaomi Redmi Note 9S Overlay
Fixed tilt issues and brightness

phhusson commented on Xiaomi/RedmiNote9S/res/values/config.xml:
```diff
@@ -0,0 +1,957 @@
+         * The remote submix module "audio.r_submix.default" must be installed on the device.
+         * The device must be provisioned with HDCP keys (for protected content).
+    -->
+     <bool name="config_enableWifiDisplay">false</bool>
```
false is already the default

phhusson commented on Xiaomi/RedmiNote9S/res/values/config.xml:
```diff
@@ -0,0 +1,957 @@
+     <bool name="config_hotswapCapable">true</bool>
+
+    <!-- Is the notification LED intrusive? Used to decide if there should be a disable option -->
+     <bool name="config_intrusiveNotificationLed">true</bool>
```
That's the default because of device/phh/treble/overlay

phhusson commented on Xiaomi/RedmiNote9S/res/values/config.xml:
```diff
@@ -0,0 +1,957 @@
+     <bool name="config_tether_upstream_automatic">true</bool>
+
+    <!-- If this is true, the screen will come on when you unplug usb/power/whatever. -->
+     <bool name="config_unplugTurnsOnScreen">true</bool>
```
This isn't device-specific and already set by default by device/phh/treble/overlay

phhusson commented on Xiaomi/RedmiNote9S/res/values/config.xml:
```diff
@@ -0,0 +1,957 @@
+         on the headphone/microphone jack. When false use the older uevent framework. -->
+     <bool name="config_useDevInputEventForAudioJack">true</bool>
+
+     <bool name="config_wifiDisplaySupportsProtectedBuffers">false</bool>
```
useless since wifi display is disabled

phhusson commented on Xiaomi/RedmiNote9S/res/values/config.xml:
```diff
@@ -0,0 +1,957 @@
+     <bool name="config_wifiDisplaySupportsProtectedBuffers">false</bool>
+
+    <!-- Boolean indicating whether the wifi chipset has background scan support -->
+     <bool name="config_wifi_background_scan_support">false</bool>
```
I have to say I'm pretty suprised by this. Can you double-check on stock rom?

phhusson commented on Xiaomi/RedmiNote9S/res/values/config.xml:
```diff
@@ -0,0 +1,957 @@
+         Specify the component name or an empty string if none.
+         Note that doze dreams are not subject to the same start conditions as ordinary dreams.
+         Doze dreams will run whenever the power manager is in a dozing state. -->
+     <string name="config_dozeComponent">com.android.systemui/com.android.systemui.doze.DozeService</string>
```
Not device-specific and already set in device/phh/treble/overlay

phhusson commented on Xiaomi/RedmiNote9S/res/values/config.xml:
```diff
@@ -0,0 +1,957 @@
+     <string name="config_dozeComponent">com.android.systemui/com.android.systemui.doze.DozeService</string>
+
+    <!-- Whether device has screen with higher aspect ratio -->
+    <bool name="config_haveHigherAspectRatioScreen">true</bool> 
```
This doesn't exist in AOSP

phhusson commented on Xiaomi/RedmiNote9S/res/values/config.xml:
```diff
@@ -0,0 +1,957 @@
+    <!-- Screen brightness used to dim the screen when the user activity
+         timeout expires.  May be less than the minimum allowed brightness setting
+         that can be set by the user. -->
+    <integer name="config_screenBrightnessDim">6</integer>
```
6 looks pretty high to me, check what's in stock rom? You'll probably need to do /4 compared to stock rom because stock is likely in 10bits, we're 8bits

phhusson commented on Xiaomi/RedmiNote9S/res/values/config.xml:
```diff
@@ -0,0 +1,957 @@
+    <string-array name="config_gpsParameters">
+        <item>SUPL_HOST=supl.qxwz.com</item>
+        <item>SUPL_PORT=7275</item>
+        <item>SUPL_VER=0x20000</item>
```
Would be nice to try is version 3 works since it's more secure

phhusson commented on Xiaomi/RedmiNote9S/res/values/config.xml:
```diff
@@ -0,0 +1,957 @@
+
+    <!-- Values for GPS configuration -->
+    <string-array name="config_gpsParameters">
+        <item>SUPL_HOST=supl.qxwz.com</item>
```
uhhhh, I really don't feel safe using "qxwz.com" for supl.
Could you check if supl.google.com still works instead?

phhusson commented on Xiaomi/RedmiNote9S/res/values/config.xml:
```diff
@@ -0,0 +1,957 @@
+    </string-array>
+
+    <!-- Doze: does this device support STATE_DOZE?  -->
+    <bool name="doze_display_state_supported">true</bool>
```
This isn't a framework-res value, this is a SystemUI value

phhusson committed:
Add a high priority overlay to force-enable gestures

This is done because vendor/google/apps/SearchLauncher/overlay disables
it behind our back.

phhusson committed:
Gestures overlay didnt have correct path, so was ignored

sooti committed:
Curtana: Fix brightness to stock values and add notch

phhusson committed:
Add non-Samsung Exynos Shannon IMS

phhusson committed:
Up treble app to 7f1b1ff

phhusson committed:
Add Moto One Action overlay (vendor is perfect except for notch)

phhusson committed:
There is now one use case of max bt audio device connected, which is Qin2Pro to 1

phhusson committed:
Allow ignoring symbol blacklist for super overlays

phhusson committed:
Fix Mi9 violation on doze component

phhusson committed:
Mark Gestures overlay as super overlay

phhusson committed:
New known keys

phhusson committed:
[Umidigi Power] No status_bar override on pie

phhusson committed:
[PNX 8.1 X7] No status_bar height override

AndyCGYan committed:
a60q: Increase status bar height in landscape mode

Change-Id: I66d5e5c2cdf06f97503392985ce3df6ed6a47482

phhusson committed:
Update treble app to 19b2a8aa

phhusson committed:
Fix wrong rebase

AndyCGYan committed:
Add resources for Samsung Galaxy A51 5G (a51x)

Change-Id: I12a8a28ad29cd83ec708f1f77ad4032fc6aec340

phhusson committed:
Update webview override to latest Google setup

Chrome Canary no longer ships with webview, only webview does

phhusson committed:
Upto treble app to fdc0f3cf0d

phhusson committed:
Add SPRD-IMS overlay for when we will have working ims on sprd

sooti committed:
Cleanup note 9s overlays and fix brightness issues

benutzer193 commented on Xiaomi/RedmiNote9S/AndroidManifest.xml:
```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.xiaomi.redminote8pro"
```
this seems to be unintentional and probably should be:

```suggestion
        package="me.phh.treble.overlay.xiaomi.redminote9s"
```

benutzer193 commented on Xiaomi/RedmiNote9ProMax/AndroidManifest.xml:
```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.xiaomi.redminote8pro"
```
probably should be:
```suggestion
        package="me.phh.treble.overlay.xiaomi.redminote9promax"
```

benutzer193 commented on Xiaomi/RedmiNote9Pro/AndroidManifest.xml:
```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.xiaomi.redminote8pro"
```
Probably should be
```suggestion
        package="me.phh.treble.overlay.xiaomi.redminote9pro"
```

benutzer193 reviewed (commented):
The packages seem to be incorrectly defined as Redmi Note 8 Pro.

sooti committed:
Fix 9s overlay names

phhusson committed:
Up treble app to 2180278b

sooti committed:
initial commit of umidigi a7 pro

AndyCGYan committed:
Add resources for Moto G8 (rav)

See comment for fixing notch behaviour - thanks @TadiT7

Change-Id: Ife970f104de10648399252c6dbad645538e55da0

phhusson committed:
Move overlays from /system/overlay to /system/product/overlay

sooti committed:
Add Doogee Y8 and fix our overlay paths

Commit:
[daisy] fix wrong statusbar height

  broken after 542ff298281f7e926f3188aabd430155e87560ce

Commit:
move Moto G8 to the product too

  fix d8ea24d

phhusson committed:
Update moto one action overlay to match on Q vendor

phhusson committed:
Include overlays for R IMS

phhusson committed:
Fix tests for Q target

phhusson committed:
Move remaining overlays to /product

phhusson committed:
Up treble app to a40382

phhusson committed:
Package name part cant begin with a digit in aapt2

sooti committed:
Switch to blue theme for redmi note 9 pro and umidigi a7 pro

sooti committed:
Curtana: Add warmup time for light sensor

sooti committed:
CAF-IMS: move to telephony and update overlay

AndyCGYan committed:
Add resources for ASUS Zenfone 5 (ASUS_X00QD, Q vendor)

Change-Id: I33f6ff5498098ebb8ebfd25b2814e94fc62474aa

phhusson committed:
Add an overlay to control the presence of dt2w setting in user settings

phhusson committed:
Update treble app to f9388

phhusson committed:
Move new overlays to /product

phhusson committed:
Fix tests

phhusson committed:
Up treble app to 1d86a11

phhusson committed:
Override tethering parameters to fix mediatek tethering

phhusson committed:
Override tethering parameters to fix mediatek tethering

PeterCxy committed:
add Spreadtrum IMS overlay for R

phhusson committed:
Up treble app to 00455029

phhusson committed:
Fix broken xml tag

phhusson committed:
Remove direct status_bar_height

phhusson committed:
Fix overlay.mk ordering

phhusson committed:
Don't test non-framework-res overlays

phhusson committed:
Up treble app to 61de6157

phhusson committed:
Up treble app to 537ccd6ca4

sooti committed:
update app

sooti committed:
update apk

sooti committed:
Remove redmi 9 notch fill overlay

sooti committed:
Fix 5ghz on 11 vendor for Note 9S & update app

sooti committed:
Merge branch 'pie' of https://github.com/phhusson/vendor_hardware_overlay into HEAD

sooti committed:
fix a7pro mismerge

sooti committed:
Merge branch 'pie' of https://github.com/phhusson/vendor_hardware_overlay into HEAD

sooti committed:
update treble app

sooti committed:
Merge branch 'pie' of https://github.com/phhusson/vendor_hardware_overlay into HEAD

sooti committed:
Redmi Note 9S: Refactor and fix missing notch with A12 vendor

sooti committed:
Merge https://github.com/phhusson/vendor_hardware_overlay into pie-rebase
